src/app/(auth)/layout.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Metadata } from 'next';
+import { Inter } from 'next/font/google';
+import '../../stylesheets/styles.scss';
+import { AuthProvider } from "../../components/providers/auth-provider";
+import AuthRedirect from "@/components/auth-redirect";
+import { Toaster } from "../../components/ui/toaster";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Auth",
+  description:
+    "Easily track and manage your personal or business expenses with our intuitive and feature-rich expense manager.",
+};
```

Comment:
`metadata` object in the layout will get applied to all the pages inside the layout. Basically you won't want that since each page may have different contents. Sometimes it makes sense to define `metadata` values in the layout like `description` here since you may want the description to be same for all auth pages, but not `title`. Login page may have title `Login to expense manager` and forgot password page may have `Forgot password`. In such cases, you can define the common metadata values on the layout and define page specific values on the page itself. There is also overriding rules for metadata values as well. More here [https://nextjs.org/docs/app/building-your-application/optimizing/metadata](https://nextjs.org/docs/app/building-your-application/optimizing/metadata)

Reply:
will resolve at the end this comment.